modules/rosa-adding-taints-cli.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+// * osd_cluster_admin/osd_nodes/osd-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-adding-taints-cli{context}"]
+= Adding taints to a machine pool using ROSA CLI
+
+You can add taints to a machine pool for your Red Hat OpenShift Service on AWS (ROSA) cluster by using the ROSA CLI.
+
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+* You created an {product-title} cluster.
+endif::[]
+* You have an existing machine pool that does not contain any taints and contains at least two instances.
+
+.Procedure
+
```

Comment:
I believe you are missing a closing `endif` for the condition you set at line 27.

**Edit**: OK. I see you close it at line 115, but I am not sure I understand the intent here. If I am following correctly, the entire procedure is to only appear in the the ROSA distro?

Reply:
Yes, that is the intention (entire procedure to only appear in the ROSA distro.